nb2workflow/service.py

```diff
@@ -691,9 +712,15 @@ def main():
     parser.add_argument('--debug', action="store_true")
     parser.add_argument('--one-shot', metavar='workflow', type=str)
     parser.add_argument('--pattern', type=str, default=r'.*')
+    parser.add_argument('-s', '--settings', action="append", default=None)
```

Comment:
Naively, I would expect this argument to be a path to `settings.toml` file. Ability to set these options from cli may be useful, though. BTW, some of existing options, like e.g. host/port may be also set in the config file

Reply:
they're now in the config file